Monika After Story/game/0config.rpy

```diff
@@ -113,6 +113,12 @@ init -1200 python:
     renpy.config.menu_clear_layers = ["front"]
     renpy.config.gl_test_image = "white"
 
+    # Remove extra save location (for whatever reason we do this)
+    # FIXME: Also, this is unsafe since renpy has a thread that
```

Comment:
should we just not do this now if its unsafe?

Reply:
We've been doing that for years, I'm not sure how bad it is, but it's something I wanted to mark as look into later. I don't think I can remove it right away either - it might mess up our auto backups.